BrainPortal/app/models/boutiques_portal_task.rb

```diff
@@ -758,8 +758,16 @@ def boutiques_module_information #:nodoc:
     integrator_modules.map do |module_name, _|
       module_name = module_name.constantize
       rev_info    = module_name::Revision_info
-      "#{rev_info.basename} rev. #{rev_info.short_commit} #{rev_info.time} (author: #{rev_info.author})"
+      rev_info.format("%f rev. %s %a %d")
```

Comment:
As far as I know this change is already in the codebase, maybe you included commits that are not related to this PR?